### PR TITLE
Convert some deposit-all numbers to bigints

### PR DESCRIPTION
**Pull request by NullSoldier**
## Summary

I noticed some currency amounts in deposit all were in numbers that should be ore. We should _always_ take strings and parse them into bigints for ore amounts.

```
Balance
Confirmed:   $IRON 0.00021458
Unconfirmed: $IRON 0.00021458
Pending:     $IRON 0.00021458

--- Completed Transactions (0) ---
```

## Testing Plan
I ran the command manually and looked at the UI.

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish-cli/src/commands/deposit-all.ts [hunk 1]
```diff
@@ -111,7 +121,7 @@ export default class DepositAll extends IronfishCommand {
       this.client,
       this.api,
       expirationSequenceDelta,
-      fee,
+      Number(fee),
```
Comment (NullSoldier) [inline ironfish-cli/src/commands/deposit-all.ts:124]: This is still broken, and all of this needs to be fixed in another PR. verifyCanSend is broken because it's accepting ore but treating it like iron.

**Review by jowparks — APPROVED** [review]
